pages/linux/rofi.md

```diff
@@ -13,3 +14,7 @@
 - Switch between windows:
 
 `rofi -show window`
+
+- Pipe a list of items to stdin and print the selected item stdout:
+
+`echo "{{Something\nOr\nOther}}" | rofi -dmenu`
```

Comment:
```suggestion
`echo -e "{{Coiche1\nChoice2\nChoice3}}" | rofi -dmenu`
```
`echo` needs `-e` to interpret newlines `\n`. Also, I think using `ChoiceN` for the items is clearer.